docs/common/istio-nftables.adoc

```diff
@@ -0,0 +1,333 @@
+// Variables embedded for GitHub compatibility
+:istio_latest_version: 1.27.2
+:istio_latest_version_revision_format: 1-27-2
+:istio_latest_tag: v1.27-latest
+:istio_release_name: release-1.27
+:istio_latest_minus_one_version: 1.27.2
+:istio_latest_minus_one_version_revision_format: 1-27-2
+
+link:../README.md[Return to Project Root]
+
+== Table of Contents
+
+* link:#istio-nftables-backend[Istio nftables backend]
+  ** link:#prerequisites[Prerequisites]
+  ** link:#installation[Installation]
+    *** link:#install-with-sail-operator-on-openshift[Install with Sail Operator on OpenShift]
+    *** link:#install-in-ambient-mode[Install in Ambient Mode]
+  ** link:#validation[Validation]
+  ** link:#Upgrade[Upgrade]
+    *** link:#upgrade-with-sail-operator-on-openshift[Upgrade with Sail Operator on OpenShift]
+    *** link:#upgrade-in-ambient-mode[Upgrade in Ambient Mode]
+
+=== Istio nftables backend
+
+This document outlines the configuration steps for the nftables backend in Istio. As the official successor to iptables, nftables offers a
+modern, high-performance alternative for transparently redirecting traffic to and from the Envoy sidecar proxy. 
+Many major Linux distributions are actively moving towards adopting native nftables support. 
+
+=== Prerequisites
+
+* *nftables version*: Requires `+nft+` binary version 1.0.1 or later.
+
+==== Installation
+
+The support for native nftables in Istio sidecar mode was implemented in the upstream istio https://github.com/istio/istio/blob/master/releasenotes/notes/nftables-sidecar.yaml[release-1.27].
+It is disabled by default. To enable it, you can set a feature flag as `+values.global.nativeNftables=true+`.
+
+===== Install with Sail Operator on OpenShift
+
+When you install IstioCNI and Istio resources with Sail Operator, you can create an instance of them with `+spec.values.global.nativeNftables=true+`. 
+This feature configures Istio to use the `+nftables+` backend instead of `+iptables+` for traffic redirection.
```

Comment:
```suggestion
When installing `Istio` and `IstioCNI` resources with the Sail Operator, you can enable nftables by setting `spec.values.global.nativeNftables=true` in the resource. This option configures Istio to use the nftables backend for traffic redirection instead of iptables.
```